README.md

```diff
@@ -248,6 +248,10 @@ multi-processor, multi-core machines
 * [Prometheus.io](https://github.com/prometheus/prometheus) An open-source service monitoring system and time series database
 * [HAProxy Exporter](https://github.com/prometheus/haproxy_exporter) Simple server that scrapes HAProxy stats and exports them via HTTP for Prometheus consumption
 
+# Testing
+* [Grai](https://github.com/grai-core/grai-io/) is a data catalog tool that integrates into your CI system exposing downstream impact testing of data changes. These tests prevent data changes which might break data pipelines or BI dashboards from making it to production.
```

Comment:
right link
https://github.com/grai-io/grai-core

Reply:
🤦 , thank you!